imports/plugins/core/orders/client/components/OrderCardFulfillmentGroupItem.js

```diff
@@ -33,33 +30,33 @@ class OrderCardFulfillmentGroupItem extends Component {
 
   render() {
```

Comment:
Looks like this class can be converted to a function, we want to move towards hooks.

Reply:
I have this being converted to a class in #5158 along with some other components from other PR's related that need to be converted. If really needed, I can move that over here, but would prefer to keep it where it is and do the class to function conversion in that one place.